src/udev/udev-node.c

```diff
@@ -447,6 +451,14 @@ static int link_update(sd_device *dev, const char *slink_in, bool add) {
                 _cleanup_free_ char *target = NULL;
                 struct stat st1 = {}, st2 = {};
 
+                if (i > 0) {
+                        usec_t delay = MIN_RANDOM_DELAY + random_u64_range(MAX_RANDOM_DELAY - MIN_RANDOM_DELAY);
```

Comment:
as discussed in that other bug, maybe do:

```
usec_t delay = MIN_RANDOM_DELAY + random_u64_range((1 << MIN(31,  i+1)) * (MAX_RANDOM_DELAY - MIN_RANDOM_DELAY));
```

i.e. grow the max delay exponentially on each invocation.